bindtomac-ifname-httpks.ks.in

```diff
@@ -34,7 +34,7 @@ check_gui_configurations ifname0 ifname1
 @KSINCLUDE@ post-lib-network.sh
 
 check_device_config_value ifname0 IPV6ADDR 2001:cafe:cafe::2/64 ipv6 address1 2001:cafe:cafe::2/64
-check_device_config_value ifname0 HWADDR 52:54:00:12:34:42 ethernet mac-address 52:54:00:12:34:40
+check_device_config_value ifname0 HWADDR 52:54:00:12:34:40 ethernet mac-address 52:54:00:12:34:40
```

Comment:
Why these changes are here? Is it a bug fixing?

Reply:
Yes, it is actually a fixup of previous commit.

Reply:
It would be great to have this in a separate commit but it would be hard to do now so feel free to ignore this.